src/components/AdminSettings/SIPBridge.vue

```diff
@@ -162,18 +163,31 @@ export default {
 			this.loading = false
 			showSuccess(t('spreed', 'SIP configuration saved!'))
 		},
+
+		updateSecret(value) {
+			this.secret = value
+		},
 	},
 }
 </script>
 
 <style lang="scss" scoped>
+h3 {
+	margin-top: 24px;
+	font-weight: 600;
+}
 
-.sip-bridge {
-	&__sip-groups-select,
-	&__shared-secret,
-	&__dialin-info {
-		width: 480px;
+.form {
+	&__textfield,
+	&__select {
+		width: 300px;
+		:deep(.vs__search) {
+			width: 100%;
+		}
```

Comment:
What is `vs__search` element? I haven't found it...

Reply:
There is a hidden input inside `NcSelect` to search through the options

Reply:
Oh, this is a class from `vue-select` library. But if it is hidden, why should it have overridden width?

Reply:
Because it adds horizontal scrolled and change height